### PR TITLE
147226409: update readme

### PR DESCRIPTION
**Pull request by seyi-adeleke**
#### What does this PR do?
This PR updates the readme
#### How should this be manually tested?
Viewing the Readme
#### Any background context you want to provide?
A good readme should provide installation and stack information.
#### What are the relevant pivotal tracker stories?
147226409
#### Screenshots (if appropriate)
none
#### Questions:
none

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,14 @@ Cards for Humanity is a fast-paced online version of the popular card game, Card
 
 Our Team
 --------
-Cards for Humanity was created at [Hack Reactor](http://www.hackreactor.com) by:
-* [Matt Silverstein](http://www.mattsilverstein.com/)
-* [Will Ngo](https://mrngoitall.net)
-* [Tyler McGinnis](http://www.tylermcginnis.com)
-* [David Gonzalez](http://www.truthyfalsy.com)
+Cards for Humanity was created at [Andela](https://www.andela.com) by:
```
Comment (ajaps) [inline README.md]: I don't agree with the wording you used here, this game wasn't created by us.  Upgraded maybe? but definitely not created

File: README.md [hunk 2]
```diff
@@ -27,12 +30,34 @@ The app is completely functional across all devices - from iPhone to full-sized
 
 ![alt-text](http://www.tylermcginnis.com/images/cfh3.png "Desktop with Cards")
 
-Our Tech Stack
+Installation
+----------
+* Ensure you have `git`, `node`, `npm` and `mongodb` correctly installed on your computer.
+* Download or clone this repo to your local computer.  Run `npm install` to install all prerequisite dependecies and frameworks.
+* Create a `.env` file. Update the `MONGOHQ_URL` field with a link to your development database.
+* Run `npm start` to start the app.
```
Comment (ajaps) [inline README.md]: I think new engineers need to start the mongodb server `sudo mongod` before running `npm start` command

**Review by ajaps — COMMENTED** [review]
I think overall you did a good job, except the little details which i feel should be adjusted.  i also noticed this branch has 4 commits (instead of 1)

File: README.md (hunk 1 above)
Comment (fokosun) [reply README.md]: Yes, true that ^^

**Review by ajaps — APPROVED** [review]
looks good

**Review by andela-Taiwo — APPROVED** [review]

**Pull request merged**